index.js

```diff
@@ -980,6 +980,48 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just the first N elements of the given collection if N is
+  //. greater than or equal to zero and less than or equal to the length
+  //. of the collection; Nothing otherwise. Supports Array, String, and
+  //. any other collection type which provides a slice method.
```

Comment:
> What about Array and String?

I like them unformatted, to match Just and Nothing. I use backticks with restraint because they add noise to the raw version and can even be distracting in the rendered version if lots of things are wrapped in little grey boxes. I think the method name should stand out here, though.